drivers/serial/Kconfig.nrfx

```diff
@@ -53,7 +53,7 @@ config UART_0_INTERRUPT_DRIVEN
 
 config UART_0_ASYNC
 	bool "Enable Asynchronous API support on port 0"
-	depends on UART_ASYNC_API
+	depends on UART_ASYNC_API && !UART_INTERRUPT_DRIVEN
```

Comment:
```suggestion
	depends on UART_ASYNC_API && !UART_0_INTERRUPT_DRIVEN
```

Reply:
Applied the suggestion. Thank you!

Reply:
Just last request, could you add the same change to other UARTE instances 1, 2 and 3?

Reply:
Do other instances still require exclusion between "UART_ASYNC_API" and "UART_INTERRUPT_DRIVEN"? Also there is no instance implementation for port 1, 2 and 3 so far, do you think it is still worth adding this conditional dependency for them?